src/components/shared/MarkerTooltipContents.js

```diff
@@ -11,6 +11,19 @@ import formatTimeLength from '../../utils/format-time-length';
 import type { TracingMarker } from '../../types/profile-derived';
 import type { MarkerPayload } from '../../types/markers';
 
+function markerDetail(key, label, value) {
```

Comment:
I think the shorter `Minor GCs` and `Slices` are much better. Without a unit we already understand it's a count -- especilly the people looking at this will know what they look at.

But that's nitpicking, so whatever works for you works for me !